sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/MinedOpinion.java

```diff
@@ -3,43 +3,56 @@
 
 package com.azure.ai.textanalytics.models;
 
+import com.azure.ai.textanalytics.implementation.MinedOpinionPropertiesHelper;
 import com.azure.core.util.IterableStream;
 
 /**
  * The {@link MinedOpinion} model.
  */
 public final class MinedOpinion {
-    private final AspectSentiment aspect;
-    private final IterableStream<OpinionSentiment> opinions;
+    private TargetSentiment target;
```

Comment:
This would be breaking, but it's ok since moving between previews ?

Reply:
Yeah, shouldn't be a problem

Reply:
It is a breaking in previews but it is ok to GA.